gitlab/_backends/protocol.py

```diff
@@ -17,6 +17,17 @@ def __init__(self, response: requests.Response) -> None: ...
 
 
 class Backend(Protocol):
+    @staticmethod
+    @abc.abstractmethod
+    def response_content(
```

Comment:
I believe this would be a breaking change as we still expose it in utils as a non-private function. In that case I'd maybe drop the wrapper entirely and not just deprecate it, as then we don't need to worry about maintaining the signature twice (https://github.com/python-gitlab/python-gitlab/pull/2488#discussion_r1633344488).

Might be worth grouping breaking changes and wait to merge them all together though for a major release.

Reply:
Well I didn't think it would be a breaking change if this is the new function and then the wrapper (with the deprecation) would call this one with keyword args.